.gitignore

```diff
@@ -113,3 +113,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#for vscode
+.vscode
```

Comment:
Please rename this Pull Request to what it does.

Reply:
Can we rename this or should I make a new PR?

Reply:
You can rename this with the edit button above.